### PR TITLE
FLW-21 ABHA v3 M1 Upgradation

### PR DESCRIPTION
**Pull request by sushant-bizbrolly**
Migration from v1/v2 to v3 of ABHA M1

### CODE REVIEW

File: app/src/main/java/org/piramalswasthya/sakhi/di/AppModule.kt [hunk 1]
```diff
@@ -28,10 +28,19 @@ import javax.inject.Singleton
 @InstallIn(SingletonComponent::class)
 object AppModule {
 
+//     Staging Base URL
     private const val baseTmcUrl =
         "https://amritdemo.piramalswasthya.org/"
 
-    private const val baseAbhaUrl = "https://healthidsbx.abdm.gov.in/api/"
```
Comment (drtechie) [inline app/src/main/java/org/piramalswasthya/sakhi/di/AppModule.kt:34]: What will happen when we deploy to prod?
Should this URl be changed before building? Can we pick up the URL from build config?

**Comment by drtechie** [comment]
@sushant-bizbrolly please remove all dead code.

**Comment by drtechie** [comment]
@coderabbitai full review

File: app/src/main/java/org/piramalswasthya/sakhi/di/AppModule.kt (hunk 1 above)
Comment (sushant-bizbrolly) [reply app/src/main/java/org/piramalswasthya/sakhi/di/AppModule.kt:34]: Yes, we have to make changes for that, will do that.

**Pull request merged**